example/ml02_distance_strategy.cpp

```diff
@@ -23,14 +26,14 @@ int main()
   point_xy p2(5.0, 0.0);
 
   // 1) This is direct call to Pythagoras algo
-  typedef strategy::distance::pythagoras<point_xy, point_xy, double> strategy1_type;
+  using strategy1_type = strategy::distance::pythagoras<double>;
```

Comment:
I'm wondering, should we use umbrella strategies in examples and documentation now?